eawx-build/Configuration/v1/BuildComponentFactory.cs

```diff
@@ -1,18 +1,26 @@
 using System;
+using System.IO.Abstractions;
 using EawXBuild.Core;
+using EawXBuild.Native;
+using EawXBuild.Tasks;
 using Microsoft.Extensions.Logging;
 
 namespace EawXBuild.Configuration.v1 {
-    public class BuildComponentFactory : IBuildComponentFactory
-    {
+    internal enum Tasks {
```

Comment:
Should be moved to its own file.
Maybe call it `TaskType`, but that's really optional.